src/cloud/architecture/starter-architecture.md

```diff
@@ -0,0 +1,135 @@
+
+The following diagram shows the hierarchical relationships of the Starter environments.
+
+![High-level view of Starter project]({{ site.baseurl }}/common/images/cloud/cloud_arch-starter.png)
+
+## Production environment {#cloud-arch-prod}
+
+The Production environment provides the source code to deploy Magento to the Cloud infrastructure that runs your public-facing Magento single and multi-site storefronts. The Production environment uses code from the `master` branch to configure and enable the web server, database, configured services, and your Magento application code.
+
+Because the production environment is read-only, you must make changes in the Integration environment and deploy across the architecture from the Integration environment to Staging, and finally to the Production environment. See [Deploy your store][] and [Site launch][].
+
+We highly recommend fully testing in your Staging environment and branch before pushing to the `master` branch which deploys to the Production environment.
+
+## Staging environment {#cloud-arch-stage}
+
+We recommend creating a branch called `staging` from `master`. The `staging` branch deploys code to the Staging environment to provide a pre-production environment to test code, modules and extensions, payment gateways, shipping, product data, and much more. This environment provides the configuration for all services to match the Production environment including Fastly, New Relic APM, and search.
+
+Additional sections in this guide provide instructions for final code deployments and testing production level interactions in a secure Staging environment. For best performance and feature testing, replicate your Production database into the Staging environment.
+
+{:.bs-callout-warning}
+We highly recommend testing every merchant and customer interaction in the Staging environment prior to deploying to the Production environment. See [Deploy your store][] and [Test deployment][].
+
+## Integration environment {#cloud-arch-int}
+
+Developers use the Integration environment to develop, deploy, and test:
+
+-  Magento application code
+-  Custom code
+-  Extensions
+-  Services
+
+You can have **one** active Integration environment. You create the Integration environment by creating a branch from the Staging branch. When you create an Integration environment, the environment name matches the branch name. An integration environment includes a web server and a database. It does not include all services, for example Fastly CDN and New Relic are not available.
+
+You can have an unlimited number of inactive branches for code storage. To access, view, and test an inactive branch, you must activate it.
+
+{% include cloud/note-enhanced-integration-envs-kb.md%}
+
+## Production and Staging technology stack {#technology}
+
+The Production and Staging environments include the following technologies. You can modify and configure these technologies through the [`.magento.app.yaml`][] file.
```

Comment:
This link is broken.

Reply:
The link definition for this does not include the period, so you will have to do something like the following:

```suggestion
The Production and Staging environments include the following technologies. You can modify and configure these technologies through the [`.magento.app.yaml`][magento.app.yaml] file.
```

Reply:
ah I understand now -- there were two occurrences, and I only fixed one of them.